Makefile

```diff
@@ -97,7 +97,19 @@ cleanup-ack-req: verify-cluster-variables
 	yq e '.cluster.region=env(CLUSTER_REGION)' -i tests/e2e/utils/ack_sm_controller_bootstrap/config.yaml
 	cd tests/e2e && PYTHONPATH=.. python3.8 utils/ack_sm_controller_bootstrap/cleanup_sm_controller_req.py
 
+verify-installation-option:
+	test $(INSTALLATION_OPTION) || (echo Please export INSTALLATION_OPTION variable; exit 1)
 
-deploy-kf-vanilla: bootstrap-ack
-	while ! kustomize build deployments/vanilla | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 30; done
-        
+verify-deployment-option:
+	test $(DEPLOYMENT_OPTION) || (echo Please export DEPLOYMENT_OPTION variable; exit 1)
+
+verify-aws-telemetry-option:
+	test $(AWS_TELEMETRY_OPTION) || (echo Please export AWS_TELEMETRY_OPTION variable; exit 1)
+
+install-ack: bootstrap-ack
+
+deploy-kf: verify-cluster-variables connect-to-eks-cluster verify-installation-option verify-deployment-option
+	cd tests/e2e && PYTHONPATH=.. python3.8 utils/kubeflow_installation.py --$(DEPLOYMENT_OPTION) --installation_option $(INSTALLATION_OPTION) --aws_telemetry_option $(AWS_TELEMETRY_OPTION) && cd -; 
+
+uninstall-kf: verify-cluster-variables connect-to-eks-cluster verify-installation-option verify-deployment-option verify-cluster-variables
```

Comment:
uninstall-kubeflow

Reply:
also no need for `&& cd -`
every line runs as a sub-command from the root dir

Reply:
maybe use `delete-kubeflow` instead of `uninstall-kubeflow`